pkg/config/loader_test.go

```diff
@@ -0,0 +1,330 @@
+package config
```

Comment:
Remember to add the copyright header, like in all the other source files in the project.